### PR TITLE
Progress Steps Line min-width fix

### PR DESCRIPTION
**Pull request by ravindra114**
### Checklist:

-   [ ] My code is tested
-   [ ] My code passes the PHPCS tests
-   [ ] I've created the npm build.
-   [ ] My code follows accessibility standards 
-   [ ] My code has proper inline documentation 
-   [ ] I've included any necessary tests 
-   [ ] I've included developer documentation 
-   [ ] I've added proper labels to this pull request

### CODE REVIEW

File: src/components/progress-steps/progress-steps.jsx [hunk 1]
```diff
@@ -93,6 +95,7 @@ const ProgressStep = ( {
 	size,
 	isLast,
 	index,
+	lineProps,
```
Comment (vrundakansara) [inline src/components/progress-steps/progress-steps.jsx:98]:
```suggestion
	lineProps = 'min-w-10',
```
These can be set as default.

**Review by vrundakansara — CHANGES_REQUESTED** [review]

File: src/components/progress-steps/progress-steps.jsx (hunk 1 above)
Comment (ravindra114) [reply src/components/progress-steps/progress-steps.jsx:98]: this is already set as default in parent component, on line 17

File: src/components/progress-steps/progress-steps.stories.jsx [hunk 2]
```diff
@@ -85,4 +113,5 @@ StackType.args = {
 	size: 'md',
 	type: 'stack',
 	currentStep: 3,
+	lineProps: 'min-w-10',
```
Comment (vrundakansara) [inline src/components/progress-steps/progress-steps.stories.jsx:116]:
```suggestion
```

File: src/components/progress-steps/progress-steps.stories.jsx [hunk 3]
```diff
@@ -77,6 +104,7 @@ IconVariant.args = {
 	type: 'inline',
 	currentStep: 4,
 	icon: <Check />,
+	lineProps: 'min-w-10',
```
Comment (vrundakansara) [inline src/components/progress-steps/progress-steps.stories.jsx:107]:
```suggestion
```

File: src/components/progress-steps/progress-steps.stories.jsx [hunk 4]
```diff
@@ -68,6 +94,7 @@ NumberVariant.args = {
 	size: 'md',
 	type: 'inline',
 	currentStep: 3,
+	lineProps: 'min-w-10',
```
Comment (vrundakansara) [inline src/components/progress-steps/progress-steps.stories.jsx:97]:
```suggestion
```

File: src/components/progress-steps/progress-steps.stories.jsx [hunk 5]
```diff
@@ -60,6 +85,7 @@ Default.args = {
 	size: 'md',
 	type: 'inline',
 	currentStep: 2,
+	lineProps: 'min-w-10',
```
Comment (vrundakansara) [inline src/components/progress-steps/progress-steps.stories.jsx:88]:
```suggestion
```

File: src/components/progress-steps/readme.md [hunk 6]
```diff
@@ -68,7 +73,7 @@ import { Plus } from 'lucide-react';
 const App = () => (
     <div>
         <h2>Progress Example</h2>
-        <ProgressSteps type="inline" variant="number" currentStep={2} size="md">
+        <ProgressSteps type="inline" variant="number" currentStep={2} size="md" lineProps="min-w-10">
```
Comment (vrundakansara) [inline src/components/progress-steps/readme.md:76]:
```suggestion
        <ProgressSteps type="inline" variant="number" currentStep={2} size="md">
```

**Review by vrundakansara — CHANGES_REQUESTED** [review]

**Review by vrundakansara — APPROVED** [review]

**Review by JelenaTakac — APPROVED** [review]
Tested.
Screenshot: https://bsf.d.pr/i/0FjgjW

**Pull request merged**